source/includes/cross-version-sync.rst

```diff
@@ -0,0 +1,3 @@
+Starting in 1.7.0, ``mongosync`` can synchronize data between clusters
+running different MongoDB major server versions. For example,
+synchronize clusters running MongoDB 6.0 and 7.0.
```

Comment:
It's strange that the [v1.2 release note ](https://preview-mongodbjasonpricemongodb.gatsbyjs.io/cluster-sync/DOCSP-29503-cross-version-sync/release-notes/1.2/)was changed and mentioning cross-version support
```
Release Notes for mongosync 1.2
Version 1.2 Released February 16, 2023

Migrating Older Versions of MongoDB
Starting in version 1.2, mongosync can sync between clusters running some older (pre-6.0) releases of MongoDB Server. This feature requires additional preparation and configuration in pre-6.0 releases.

Also, mongosync can perform a cross version migration from a pre-6.0 MongoDB Server to a later version, which also requires additional preparation.

If you would like to migrate pre-6.0 versions of MongoDB Server using mongosync, or perform a cross version migration, please 
[contact](https://mongodb.com/contact)
 your account team to inquire about Professional Services.
```


But the pre-6.0 cross-version support was not mentioned in v1.7 release note